src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -215,8 +216,11 @@
     <value>"{0}" could not be installed.
 {1}.</value>
   </data>
-  <data name="InstallHelp" xml:space="preserve">
-    <value>Installs a source or a template pack.</value>
+  <data name="OptionDescriptionInstall" xml:space="preserve">
+    <value>Installs a template pack from the path or using NuGet ID provided.
+To install the package from the NuGet feed, use the NuGet package identfier. Use "package-identifier::version" to specify the version to install. If a version of the template was already installed when you run this command, the template will be updated to the specified version, or to the latest stable version if no version was specified.
+To install the package from the file system, specify the path to the NuGet package file.
```

Comment:
Would it help to mention the expected file extension here?

Reply:
I think it would be helpful, added.